### ISSUE TITLE
Sortable Data Grid in German announced as "sort in ascending order" instead of "sorted in ascending order"

### ISSUE THREAD

**Issue by myrpfl** [FreedomScientific/VFO-standards-support#892]
In German, when a sortable header cell of a data grid is being focused, the current sorting status via aria-sort should be announced, e.g. "Aufsteigend sortiert".
Instead, JAWS reads "Aufsteigend sortieren", which means "to sort in ascending order".
This can be confusing for users, since it makes you think it will be sorted in ascending order when you press Enter. But instead it, is currently sorted in ascending order and when you press Enter, it will be sorted in descending order.

It can be tested with "Example 2: Sortable Data Grid With Editable Cells" here: https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/data-grids/
